README.md

```diff
@@ -100,6 +100,49 @@ You can also use `LocalGitHubOAuthenticator` to map GitHub accounts onto local u
 
 You can use your own Github Enterprise instance by setting the `GITHUB_HOST` environment variable.
 
+### GitHub-specific features
+
+Additional features are turned on by overriding the authenticator's
+`scope` list with particular GitHub scopes:
+
+https://developer.github.com/apps/building-integrations/setting-up-and-registering-oauth-apps/about-scopes-for-oauth-apps/
+
+Some example scopes you might want:
```

Comment:
Create a new section.

## Example GitHub scopes

The following GitHub scopes may be suitable for certain use cases: